Projects/App/Sources/Screen/TabView/HGTabView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  HGTabView.swift
+//  HGDesignSystem
+//
+//  Created by iOS신상우 on 6/15/25.
+//
+
+import SwiftUI
+import HGDesignSystem
+
+@Observable
+class HGTabViewManager {
+    fileprivate var hiddenTabbar: Bool = false
```

Comment:
fileprivate도 파일 내에 같이있다면
setTabBarHidden(true)과
hiddentTabbar = true를 써서 같이 수정이 이뤄질거같아요

```suggestion
    private(set) var hiddenTabbar: Bool = false
```
처럼 외부에서 읽기만 가능하도록 의도하고싶었어여!